src/sage/groups/affine_gps/group_element.py

```diff
@@ -138,6 +140,14 @@ def __init__(self, parent, A, b=0, convert=True, check=True):
                 raise TypeError('b must be an element of ' + str(parent.vector_space()))
             parent._element_constructor_check(A, b)
         super().__init__(parent)
+        if not A.is_immutable():
+            from copy import copy
```

Comment:
maybe rather import copy globally in the file ?

Reply:
Indeed, that is better. Done.